tests/baselines/reference/decrementOperatorWithAnyOtherTypeInvalidOperations.js

```diff
@@ -131,9 +131,15 @@ var ResultIsNumber30 = obj1.y--;
 // miss assignment operators
 --ANY2;
 ANY2--;
---ANY1--;
---ANY1++;
-++ANY1--;
---ANY2[0]--;
---ANY2[0]++;
-++ANY2[0]--;
+--ANY1;

```

Comment:
Yea I saw this and have some debut with myself as well. So this is the result of ES7 moving prefixUnaryOperation (++, --) as part of IncrementExpression. We can maintain the old behavior though it will not be consistent with the ES7 grammar